itou/eligibility/models/common.py

```diff
@@ -185,7 +189,25 @@ def is_certifiable(self):
         return self.kind in self.CAN_BE_CERTIFIED_KINDS
 
 
+class SelectedAdministrativeCriteriaQuerySet(models.QuerySet):
+    def with_is_considered_certified(self, hiring_start_at=None):
+        if not hiring_start_at:
+            # could be:
+            # is_certified = models.Q(certification_period__contains=timezone.now())
+            # but not validated by UX for the moment.
+            is_certified = models.Value(False)
+        else:
+            validity_period = InclusiveDateRange(
+                hiring_start_at - datetime.timedelta(days=self.model.CERTIFICATION_GRACE_PERIOD_DAYS),
+                hiring_start_at,
+            )
+            is_certified = models.Q(certification_period__overlap=validity_period, certified=True)
+        return self.annotate(is_considered_certified=is_certified)
+
+
 class AbstractSelectedAdministrativeCriteria(models.Model):
+    CERTIFICATION_GRACE_PERIOD_DAYS = 90
```

Comment:
```suggestion
    CERTIFICATION_GRACE_PERIOD_TIME = relativedelta(months=3)
```

?

Je suis partagé…
On gère les pass en jours (ça fait sens avec les suspensions et prolongations de manipuler une quantité absolue), mais ici c'est une constante de 3 mois, ça me titille et ça rejoint un autre retour de Joffrey et c'est potentiellement sujet à plusieurs interprétations différentes. Est-ce problématique que cette durée soit relative et puisse constituer de 89 à 92 jours ?

Reply:
C’est assez injuste à mon avis. Une durée exacte est plus facile à manipuler pour nous, et égalitaire pour tous. Souvent, on précise 3 mois (90 jours) pour expliquer les petits écarts.

Reply:
La discussion [est ouverte avec le métier](https://www.notion.so/plateforme-inclusion/Interface-certification-BRSA-10ae8fa5c35b8010a0adf2c92ae26316?p=120e8fa5c35b80e3bdf6ef3cd4c95bea&pm=s).

Reply:
Validé à 92 jours